hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -577,6 +578,10 @@ public T feeUsd(double price) {
         return self();
     }
 
+    public T signedByPayerAnd(String... keys) {
+        return signedBy(Stream.concat(Stream.of(DEFAULT_PAYER), Stream.of(keys)).toArray(String[]::new));
```

Comment:
We're creating three streams and an array just to add one item to an array?
This could be done with a single allocation and a single array copy.
```suggestion
        final String[] copy = new String[keys.length + 1];
        copy[0] = DEFAULT_PAYER;
        System.arrayCopy(keys, 0, copy, 1, keys.length);
        return signedBy(copy);
```

Reply:
Changed as suggested.